index.html

```diff
@@ -561,6 +665,38 @@
                 });
             }
 
+            function loadComponentsList() {
+                httpGet("/rest/product/" + bzProduct + "?type=enterable&include_fields=components", function(response) {
+                    var components = response.products[0].components;
+                    components.sort(function(a, b) {
+                        return a.name.localeCompare(b.name);
+                    });
+                    components.forEach(function(component) {
+                        if (!(component.is_active)) { return }
+                        var opt = document.createElement('option');
+                        opt.innerText = component.name;
+                        opt.value = component.name;
+                        document.getElementById("textComponent").appendChild(opt);
+                    });
+                });
+            }
+
+            function loadVersionsList() {
+                httpGet("/rest/product/" + bzProduct + "?type=enterable&include_fields=versions", function(response) {
+                    var versions = response.products[0].versions;
+                    versions.sort(function(a, b) {
+                        return a.name.localeCompare(b.name);
+                    });
+                    versions.forEach(function(version) {
+                        if (!(version.is_active)) { return }
+                        var opt = document.createElement('option');
+                        opt.innerText = version.name;
+                        opt.value = version.name;
+                        document.getElementById("textVersion").appendChild(opt);
+                    });
+                });
+            }
+
```

Comment:
Not sure how to reproduce it, but I once saw the same component names and versions listed twice in the select boxes in the modal. Just FYI.